python/metric_pipeline_utils/filtermatches.py

```diff
@@ -26,7 +26,7 @@ def filterMatches(matchedCatalog, snrMin=None, snrMax=None,
         astromCalibStars = False
 
     matchedCat = GroupView.build(matchedCatalog)
-    magKey = matchedCat.schema.find('slot_PsfFlux_mag').key
+    magKey = matchedCat.schema.find('base_PsfFlux_mag').key
```

Comment:
Can you briefly comment on the rationale of this change, partly for my own understanding?

Reply:
It shouldn't make any difference, but it was actually a mistake. I'll return it to its previous form.

Although now that I look at it, we use "base_" rather than "slot_" for most other quantities...